### DISCUSSION TITLE
Turning off [oh-my-zsh] Random theme loaded

### DISCUSSION THREAD

**Discussion by dacadla** [ohmyzsh/ohmyzsh#10461] [Help] [answered]
Hello, I was just wondering if you can turn off the "[oh-my-zsh] Random theme loaded". It's not a big deal, I was just wondering if i could turn it off.

**Comment by mcornella** [comment, marked as answer]
You can use `ZSH_THEME_RANDOM_QUIET=true`. That needs to go in your zshrc file, before Oh My Zsh is sourced. See the [Settings](https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#zsh_theme_random_quiet) page of the wiki for more information.
